packages/local/.env.template

```diff
@@ -1 +1 @@
-COMMERCE_PROVIDER=local
\ No newline at end of file
+COMMERCE_PROVIDER=@vercel/commerce-local
```

Comment:
Can you update all the .env.template files to be consistent across all providers